### PR TITLE
[AC-1336] Indicate unpaid subscriptions on payment screen

### PR DESCRIPTION
**Pull request by cturnbull-bitwarden**
## Type of change

```
- [ ] Bug fix
- [X] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective
When a subscription has a state of "unpaid," some text will now appear to say that "Your payment method will be charged for any unpaid subscriptions."

## Code changes

- **locales\en\messages.json:** Add unpaid subscriptions charge message
- **payment-method.component.ts:** Add subscription status to payment method component
- **payment-method.component.html:** Add warning message for unpaid subscriptions

## Screenshots

image

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Review by cyprain-okeke — COMMENTED** [review]
it good look

File: apps/web/src/locales/en/messages.json [hunk 1]
```diff
@@ -2040,6 +2040,9 @@
       }
     }
   },
+  "paymentChargedWithUnpaidSubscription": {
```
Comment (cyprain-okeke) [inline apps/web/src/locales/en/messages.json:2043]: Please confirm his message with product team

**Review by cyprain-okeke — DISMISSED** [review]

File: apps/web/src/app/billing/settings/payment-method.component.ts [hunk 2]
```diff
@@ -84,12 +88,22 @@ export class PaymentMethodComponent implements OnInit {
     if (this.forOrganization) {
       const billingPromise = this.organizationApiService.getBilling(this.organizationId);
       const orgPromise = this.organizationApiService.get(this.organizationId);
+      const subPromise = this.organizationApiService.getSubscription(this.organizationId);
```
Comment (eliykat) [inline apps/web/src/app/billing/settings/payment-method.component.ts:91]: `OrganizationSubscriptionResponse` is a subclass of `OrganizationResponse`, so this should be able to replace the `orgPromise` call.

**Review by eliykat — CHANGES_REQUESTED** [review]

File: apps/web/src/app/billing/settings/payment-method.component.ts (hunk 2 above)
Comment (cturnbull-bitwarden) [reply apps/web/src/app/billing/settings/payment-method.component.ts:91]: Nice suggestion, I just pushed an update cleaning this up

**Review by eliykat — APPROVED** [review]
Nicely done

**Pull request merged**